src/Sulu/Bundle/AdminBundle/Resources/js/containers/FieldBlocks/FieldBlocks.js

```diff
@@ -351,9 +351,16 @@ class FieldBlocks extends React.Component<FieldTypeProps<Array<BlockEntry>>> {
             return;
         }
 
+        const newFormStoreData = {};
```

Comment:
Would it not be better to instant for every new settings a new form store here with the correct data:

```
const {formInspector} = this.props;
this.blockSettingsFormStore = memoryFormStoreFactory.createFromFormKey(
     this.settingsFormKey,
     this.value[index][SETTINGS_KEY] || {},
     formInspector.locale,
     undefined,
     formInspector.options
);
```

For me it feels just false to share the same instance of FormStore between different blocks, just afraid that we again share some state between them which we maybe don't want.